cli/ssh.go

```diff
@@ -53,21 +52,14 @@ func (r *RootCmd) ssh() *clibase.Cmd {
 		Use:         "ssh <workspace>",
 		Short:       "Start a shell into a workspace",
 		Middleware: clibase.Chain(
-			r.UseClient(client),
+			r.InitClient(client),
+			clibase.RequireNArgs(1),
```

Comment:
These should perhaps be flipped? It’s better if validation is immediate and not delayed by client init.